### ISSUE TITLE
tmva/sofie_parsers does not build with latest Protobuf (22.x)

### ISSUE THREAD

**Issue by jan-busa** [root-project/root#12770]
When building latest ROOT (6.28.02) with latest Protobuf 22.4 tmva/sofie_parsers build fails during linking step with errors of the type:
```
x86_64-unknown-linux-gnu/bin/ld: ... undefined reference to `absl::lts_20230125::log_internal::LogMessage::OstreamView::~OstreamView()...
x86_64-unknown-linux-gnu/bin/ld: ... undefined reference to `absl::lts_20230125::Mutex::~Mutex()'
x86_64-unknown-linux-gnu/bin/ld: ... undefined reference to `utf8_range::IsStructurallyValid(...
```
The problem seems to be caused by adding new dependency of Protobuf on Abseil in Protobuf 22.x (https://protobuf.dev/news/2022-08-03/). To make the code build I needed to:
Add in file `cmake/modules/SearchInstalledSoftware.cmake`  after line 
```
  find_package(Protobuf)
```
lines (probably this should be dependent on version of Protobuf)
```
  find_package(absl REQUIRED)
  find_package(utf8_range REQUIRED)
```
In file `tmva/sofie_parsers/CMakeLists.txt` add inside linking after line
```
    ${Protobuf_LIBRARIES}
```
lines
```
    absl::str_format_internal
    absl::log_internal_message
    absl::hash
    absl::flat_hash_set
    absl::log_internal_check_op
    utf8_range::utf8_validity
```

**Comment by jan-busa** [comment]
With the latest abseil (20230802.0) three new libraries need to be added during linking
```
absl::log_internal_conditions
absl::status
absl::statusor
```

**Comment by dpiparo** [comment]
Dear @jan-busa , @lmoneta : it is not clear to me from the conversation if this is still an issue. If not, can this be closed?

**Comment by jan-busa** [comment]
Dear @dpiparo, I haven't checked for a while but it seems, that this issue has been fixed by #14144. I will test it and let you know later today/tomorrow in the morning.

**Comment by jan-busa** [comment]
Sorry, it took so long. As I see, the issue has been fixed in master, but still persists in the latest ("release") versions 6.28.12 and 6.30.04 so I'm not sure, should it be closed or no. For me both variants are OK.
